games/hangman.c

```diff
@@ -0,0 +1,292 @@
+/**
+ * @hangman.c
+ * @brief C implementation of hangman game. Wikipedia: https://en.wikipedia.org/wiki/Hangman_(game)
+ * @details
+ * Simple, readable version of hangman.
+ * Changed graphic to duck instead of traditional stick figure (same number of guesses).
+ * @author [AtlantaEmrys2002](https://github.com/AtlantaEmrys2002)
+*/
+
+#include <ctype.h> /// for main()
+#include <stdio.h> /// for main(), new_word(), new_guess(), won()
+#include <stdlib.h> /// for all functions
+#include <string.h> /// for main()
+#include <time.h> /// for new_game()
+#include <assert.h> /// for self-testing
+
+// global variable - word to guess
+char current_word[30] = "";
```

Comment:
Global variables should be avoided as most as possible.
Please use this as a parameter passed by reference or in a structure.